x-pack/plugins/secops/server/graphql/suricata/resolvers.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SourceResolvers } from '../../../common/graphql/types';
+import { AppResolvedResult, AppResolverOf } from '../../lib/framework';
+import { Suricata } from '../../lib/suricata';
+import { SuricataRequestOptions } from '../../lib/suricata/types';
+import { Context } from '../../lib/types';
+import { parseFilterQuery } from '../../utils/serialized_query';
+import { QuerySourceResolver } from '../sources/resolvers';
+
+// interface QuerySuricataResponse extends SourceResolvers.GetSuricataEventsArgs {
```

Comment:
commented-out code should generally not be checked-in

Reply:
👍